### DISCUSSION TITLE
How can I research my model?

### DISCUSSION THREAD

**Discussion by Lorenzobattistela** [explosion/spaCy#11633] [Help: Model Advice] [answered]
I am new to machine learning and spacy, and I am working in a project where my task is to build a custom NER. I researched spacy and I got it working, but I wonder if there is any documentation / paper where I can find more about the methods and math specifically for NER. I found some processes explanation in the docs, but I am not sure if it serves NER models too. I am basically trying to understand how NER models work step by step. If anyone knows resources, i will be grateful.

**Comment by Lorenzobattistela** [comment, marked as answer]
For anyone struggling with this, I found an extremely good article (on spacy docs) that tells me what I need to know:
https://explosion.ai/blog/deep-learning-formula-nlp

**Reply by rmitsch** [reply]
Glad you found the article to be helpful. For posterity, this [thread](https://github.com/explosion/spaCy/discussions/11471#discussioncomment-3609295) links to some resources on NER as well.
